src/etc/inc/interfaces.inc

```diff
@@ -3865,6 +3865,10 @@ function interface_dhcpv6_configure($interface = "wan", $wancfg) {
 		return;
 	}
 
+	if (file_exists("/conf/dhcp6c_duid")) { // Using user defined DUID.
```

Comment:
You should never hardcode /conf, instead you need to use (from /etc/inc/globals.inc) global var $g['conf_path']